cmake/FetchDependencies.cmake

```diff
@@ -2,10 +2,19 @@
 ### Required Dependencies ###
 #############################
 
+include(FetchContent)
+
 # Vulkan
 find_package(Vulkan REQUIRED)
 
-include(FetchContent)
+# Vulkan Memory Allocator
+if(EXISTS "${Vulkan_INCLUDE_DIR}/vma/vk_mem_alloc.h")
+    message(STATUS "Using VMA from Vulkan SDK: ${Vulkan_INCLUDE_DIR}")
+    set(VMA_INCLUDE_DIR ${Vulkan_INCLUDE_DIR})
```

Comment:
The FindVulkan CMake module uses such goofy casing

Reply:
I know. The rules for find modules are to name the script FindFoo.cmake and to output variables in the form Foo_VARIABLE. Since package names tend to not be all caps, you see this everywhere.